### DISCUSSION TITLE
What is dt in the

### DISCUSSION THREAD

**Discussion by bho01** [skylinecc/fdg#19] [Q&A]
In the config, I see the scale, cooling factor, and dt.
In my research into fructherman-reingold's algorithm, my understanding was that dt was was cooling factor. Was wondering what dt was.

By default it is set to 0.35, but going up to 0.2 it doesn't seem to do much in my graph

**Comment by grantshandy** [comment]
`dt` is the delta time, or the general "step size" that the algorithm uses when applying the force. This exists because `FruchtermanReingold` applies the forces using the [Euler method](https://en.wikipedia.org/wiki/Euler_method). Practically, reducing it will improve accuracy but slow it down and making it larger will reduce accuracy but speed it up.
